src/server/tweetFinder.js

```diff
@@ -3,16 +3,19 @@ import { TweetBuilder, TweeterBuilder } from '../shared/data/databaseObjects';
 import { newPromiseChain, flattenImmutableObject } from '../shared/utilities';
 import { searchAndSaveFromTwitter } from './twitterSearch';
 
+export const MAX_TWEET_RESULTS = 300;
+
 /**
  * Searches our database for Tweets and returns them.
- * If our search is not good enough, access Twitter directly to retrieve more tweets.
+ * May search the Twitter API directly for new results.
```

Comment:
Can you make the comment more explicit, maybe even change potentiallySearchTwitter to take in 2 paramaters: searchTerms(body.searchTerms) and shouldSearchTwitter(body.searchTwitter)